compose.py

```diff
@@ -314,10 +314,10 @@ def create_argmap(images):
         help=
         'Use specified Docker image to generate Docker image. Specified as <image-name>,<full-image-name>. <image-name> can be "min" or "full". Both "min" and "full" need to be specified at the same time. This will override "--container-version".'
     )
-    parser.add_argument('--enable-gpu',
```

Comment:
We need to keep the --enable-gpu flag for backward compat. We want it to be a flag that can be set to true or false but default to true. As cpu-only build is not supported the flag isn't needed and we don't want to document it. So set the appropriate setting so that it does not appear in the --help output.